spec/features/idv/in_person_spec.rb

```diff
@@ -6,6 +6,10 @@
 
   before do
     allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
+    allow(IdentityConfig.store).to receive(:idv_api_enabled_steps).and_return(
+      ['password_confirm',
+       'personal_key', 'personal_key_confirm'],
```

Comment:
```suggestion
      ['password_confirm', 'personal_key', 'personal_key_confirm'],
```
I think it'd read better either like this or with one value per line

Reply:
Fixed in 945454348626ae9a2c46d75db05f99301b917db3